### PR TITLE
adding beziersurgeon

### PR DESCRIPTION
**Pull request by connordavenport**

### CODE REVIEW

File: _data/BezierSurgeon.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: An extension for precision point insertion.
+developer: Connor Davenport
+developerURL: http://www.connordavenport.com
+extensionName: BezierSurgeon
+extensionPath: BezierSurgeon.roboFontExt
+repository: https://github.com/connordavenport/BezierSurgeon
+icon: https://raw.githubusercontent.com/connordavenport/BezierSurgeon/master/images/mechanicIcon.png
+tags: [interpolation, drawing]
+dateAdded: 2019-06-15 16:03:24
```
Comment (typemytype) [inline _data/BezierSurgeon.yml:9]: could you change the date to today :)

Comment (connordavenport) [reply _data/BezierSurgeon.yml:9]: Oops sorry, I copied the Theme Manager file😜 fixing now

Comment (typemytype) [reply _data/BezierSurgeon.yml]: super!

**Comment by connordavenport** [comment]
All fixed:)

**Pull request merged**